packages/track-event-mixin/track-event-gesture.ts

```diff
@@ -0,0 +1,308 @@
+// Check for touch-only devices
+const IS_TOUCH_ONLY = navigator.userAgent.match(/iP(?:[oa]d|hone)|Android/);
```

Comment:
The regex here is quite obfuscated for no good reason imo. I'd prefer the following
```suggestion
const IS_TOUCH_ONLY = navigator.userAgent.match(/iPod|iPad|iPhone|Android/);
```

Reply:
Updated to use `isIOS` from browser-utils package instead.

Reply:
Ok. Though now it doesn't match Android anymore, is that intended?

Now it's using:
```js
const isIOS =
  /iPad|iPhone|iPod/.test(navigator.platform) ||
  (navigator.platform === 'MacIntel' && navigator.maxTouchPoints > 1);
```

I tried running the tests locally on Android by running `yarn test:debug` and loaded `my-ip:9876/debug.html` on my phone. At least all track event mixin tests still passed (however one failed test with `autofocus` and one with usage statistics you may want to check separately).

Reply:
I just noticed your other comment about testing manually with Android later. I guess it's fine then.

Reply:
Right, in this file this `isIOS` is only used in context of this comment:
```js
// don't add mouse handlers on iOS because they cause gray selection overlays
```
So I think it's good.